internal/datacoord/services.go

```diff
@@ -456,6 +456,39 @@ func (s *Server) GetCollectionStatistics(ctx context.Context, req *datapb.GetCol
 	}
 	nums := s.meta.GetNumRowsOfCollection(ctx, req.CollectionID)
 	resp.Stats = append(resp.Stats, &commonpb.KeyValuePair{Key: "row_count", Value: strconv.FormatInt(nums, 10)})
+
```

Comment:
this stats is used to know the percentage of segment which has been backfilled and advanced to correct schema version